### DISCUSSION TITLE
How do backup from one Citus cluster and restore to another?

### DISCUSSION THREAD

**Discussion by YMikheev** [citusdata/citus#7365] [Q&A]
We are trying to use pg_dump/pg_restore, but it doesn't work. I think because of metadata difference between Citus coordinator and Postgress. Perhaps there is another utility?

**Comment by tejeswarm** [comment]
@YMikheev pg_dump/restore should work, what was the issue you are seeing using pg_dump/pg_restore ? Are you dumping, restore and then distributing the data? If you can detail the steps you are taking that will help

In future, we will add the support for Citus-to-Citus directly in pgcopydb
https://github.com/dimitri/pgcopydb

**Reply by YMikheev** [reply]
@tejeswarm
As a result, we managed to do backup/restore using pg_dump/pg_restore on a Citus. It wasn't easy.
The speed of archiving and recovery is quite low.

When do you plan to add the support for Citus-to-Citus directly in pgcopydb?
pgcopydb look interesting for as.

And, do you plan to add schema compare for Citus?

**Reply by YMikheev** [reply]
@tejeswarm do you now real cases of using pgcopydb for Citus-to-Citus?
